pkg/operator/ceph/object/controller.go

```diff
@@ -75,13 +75,15 @@ var controllerTypeMeta = metav1.TypeMeta{
 
 // ReconcileCephObjectStore reconciles a cephObjectStore object
 type ReconcileCephObjectStore struct {
-	client              client.Client
-	bktclient           bktclient.Interface
-	scheme              *runtime.Scheme
-	context             *clusterd.Context
-	clusterSpec         *cephv1.ClusterSpec
-	clusterInfo         *cephclient.ClusterInfo
-	objectStoreChannels map[string]*objectStoreHealth
+	client                   client.Client
+	bktclient                bktclient.Interface
+	scheme                   *runtime.Scheme
+	context                  *clusterd.Context
+	clusterSpec              *cephv1.ClusterSpec
+	clusterInfo              *cephclient.ClusterInfo
+	objectStoreChannels      map[string]*objectStoreHealth
+	rgwAdminOpsUserAccessKey string
```

Comment:
Can the key and secret be on the ReconcileCephObjectStore struct? They are different per object store, right? I believe this struct is used for reconciling all object stores?